initializer/mount.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package main
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+
+	"github.com/edgelesssys/contrast/internal/logger"
+	"github.com/edgelesssys/contrast/internal/mount"
+	"github.com/spf13/cobra"
+)
+
+// luksVolume struct holds the representative attributes related to a LUKS encrypted volume.
+type luksVolume struct {
+	devicePath           string
+	mappingName          string
+	volumeMountPoint     string
+	encryptionPassphrase string
+}
+
+func must(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+// NewSetupEncryptedMountCmd creates a Cobra subcommand of the initializer to set up the specified encrypted LUKS volume.
+func NewSetupEncryptedMountCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "setupEncryptedMount -d [device-path] -m [mount-point]",
```

Comment:
Can we make the subcommand name a bit simpler, e.g. `cryptsetup` or just `luks`? That would also elegantly circumnavigate the style question of snake/kebab/camel-case.